tests/tests/tier0/bluechi-agent-logtarget/test_agent_logtarget.py

```diff
@@ -0,0 +1,80 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+import pytest
+from typing import Dict
+
+from bluechi_test.test import BluechiTest
+from bluechi_test.container import BluechiControllerContainer, BluechiNodeContainer
+from bluechi_test.config import BluechiControllerConfig, BluechiNodeConfig
+
+NODE_GOOD = "node-good"
+NODE_WITH_LONG_LOGTARGET = "node-long-logtarget"
+NODE_WITH_NOT_VALID_VALUE = "node-with-not-valid-value"
+NODE_WITH_NUMBERS_ONLY_IN_LOGTARGET = "node-numbers-only"
+
+
+def start_with_invalid_logtarget(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer]):
+    _, output = nodes[NODE_GOOD].exec_run('systemctl status bluechi-agent')
+    output_systemd = str(output)
+    if "fail" in output_systemd.lower():
+        raise Exception(f"{NODE_GOOD} bluechi-agent should NOT failed during the start of the service")
```

Comment:
Could you replace those lines by
```python
node_good = nodes[NODE_GOOD]
assert node_good.wait_for_unit_state_to_be("bluechi-agent", "active")
```
similar to https://github.com/containers/bluechi/pull/523/commits/ad1d45750674f3c46bf56adf9b17e530d804c159. It reduces amount of (repeated) code and is more stable since it'll wait for the expected state for n seconds.

Reply:
oh, we need to update all sources, looks like I pull the one that doesn't use the most recent approach.

Reply:
Good point. Could you fix the other code locations as well? :)

Reply:
sure, I will prepare another PR for that.